config/config.go

```diff
@@ -64,6 +64,10 @@ type Config struct {
 	LogLevel zerolog.Level
 	// LogWriter defines the writer used for logging
 	LogWriter io.Writer
+	// RateLimit requests made by the client identified by IP over any protocol (ws/http).
+	RateLimit uint64
```

Comment:
Is this supposed to be in this PR?
nothing else mentions it

Reply:
not really, it's needed in the follow up PR https://github.com/onflow/flow-evm-gateway/pull/267 
Which I will get in as soon as I validate this one.